Resources/doc/index.rst

```diff
@@ -0,0 +1,22 @@
+==============
+Filter manager
+==============
+
+Filter manager for product lists. It provides ties between commonly used filtering options and UI elements with Elasticsearch repositories.
+You can use it from a single controller.
+
+Documentation for ONGR Filter Manager Bundle can be found `here <setup.rst>`_.
+
+`Read the documentation for 0.1.x (master) <https://github.com/ongr-io/FilterManagerBundle/blob/master/Resources/doc/index.rst>`_
+
+-------
+License
+-------
+
+This bundle is under the MIT license. Please, see the complete license in the bundle `LICENSE` file.
```

Comment:
This bundle is covered by the MIT license. Please see the complete license in the bundle `LICENSE` file.